client/gtk2/ibusimcontext.c

```diff
@@ -1529,6 +1530,69 @@ _set_cursor_location_internal (IBusIMContext *ibusimcontext)
 
     area = ibusimcontext->cursor_area;
 
+#if GTK_CHECK_VERSION (3, 98, 4)
+    graphene_point_t p;
+
+    if (!gtk_widget_compute_point (ibusimcontext->client_window,
+                                   GTK_WIDGET (gtk_widget_get_root (ibusimcontext->client_window)),
```

Comment:
It would beyond 80 char columns and I think it's good to create another  variable `root = GTK_WIDGET (gtk_widget_get_root (ibusimcontext->client_window));`.